packages/app-root/src/app/users/[login]/page.js

```diff
@@ -0,0 +1,7 @@
+export default function UserPage({ params }) {
+  return (
+    <div>
+      <p>This is {params.login}'s profile page.</p>
```

Comment:
Open to suggestion about the dynamic segment name, related options:
1. currently it's `login` per the panoptes user resource property `login` (maybe should be `user_login`)
2. in [PFE the similar route](https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/router.jsx#L151) uses `:profile_name`
3. another option is `username` or other similar variations